pkg/provider/handlers/deploy.go

```diff
@@ -247,7 +247,7 @@ func prepareEnv(envProcess string, reqEnvVars map[string]string) []string {
 }
 
 func getMounts() []specs.Mount {
-	wd, _ := os.Getwd()
+	wd, _ := os.LookupEnv("hosts_dir")
```

Comment:
I would suggest that we need a fallback for when the string is empty.

Could you implement that please?

`os.LookupEnv` should help here.